### ISSUE TITLE
Error with istio pods when upgrading the cluster

### ISSUE THREAD

**Issue by Syerram** [cloudfoundry/cf-for-k8s#385]
## Describe the bug
When upgrading the cf-for-k8s with a new istio version, Kubernetes throws an error for few isito components.
```
evicting pod istio-system/istio-policy-57c6694d74-hh4l2
error when evicting pod "istio-policy-57c6694d74-hh4l2" (will retry after 5s): Cannot evict pod as it would violate the pod's disruption budget.
```
For additional info, see this [slack thread](https://cloudfoundry.slack.com/archives/CH9LF6V1P/p1597316375044600)

## Expected behavior
I should be able to upgrade the cluster and istio should be come up without issues

## Additional context
Networking believes the issue is due to a single replica instead of a minimum of 2 replicas to support upgrades.

## Workaround
Manually delete the istio pods.

**Comment by KauzClay** [comment]
For visibility, here is the story for the cf-k8s-networking work to address this issue: [#174326841](https://www.pivotaltracker.com/n/projects/2407973/stories/174326841)

**Comment by jamespollard8** [comment]
Fixed by the networking team in [#174326841](https://www.pivotaltracker.com/n/projects/2407973/stories/174326841)
